main.go

```diff
@@ -3,48 +3,51 @@ package main
 import (
 	"os"
 
-	v2log "github.com/bitrise-io/go-utils/v2/log"
+	"github.com/bitrise-io/go-utils/v2/log"
+	"github.com/bitrise-steplib/steps-xcode-build-for-test/step"
 )
 
 func main() {
 	os.Exit(run())
 }
 
 func run() int {
-	logger := v2log.NewLogger()
+	exitCode := 0
 
-	s := createStep(logger)
-	cfg, err := s.ProcessConfig()
+	logger := log.NewLogger()
+	xcodebuildBuilder := createXcodebuildBuilder(logger)
+
+	cfg, err := xcodebuildBuilder.ProcessConfig()
 	if err != nil {
 		logger.Errorf("Process config: %s", err)
-		return 1
+
+		exitCode = 1
+		return exitCode
```

Comment:
Is this `exitCode` local var necessary? I was just comparing it to the previous state and found that at the first look the old one was more readable for me. 

When I first read the `exitCode := 0` assignment at the top I was already curious what values this will take. Then run in the the 
```
exitCode = 1
return exitCode
```
which did me think about it why it is like this. 

Then the `exitCode` is set to 1 if there is an error either in the run on export phase and returned. But if there is no error then I had to go back and follow one more time what value the `exitCode` will hold. 

I only write it down this detailed to highlight why for me the old version was coming with less cognitive load. But this is also subjective. 😄

Reply:
I do not have a strong opinion on this, but others thought it is better with the variable: https://github.com/bitrise-steplib/bitrise-step-xcode-test-without-building/pull/1#discussion_r889262274